symbiont/middleware/UserAuthVerify.py

```diff
@@ -4,31 +4,57 @@
 from firebase_admin import auth
 from starlette.middleware.base import BaseHTTPMiddleware
 
-
 class AuthTokenMiddleware(BaseHTTPMiddleware):
     """
     Middleware for verifying and extracting user authentication token from the Authorization header.
+
+    Attributes:
+        ROUTES_TO_EXCLUDE (list): List of routes to exclude from authentication.
     """
 
     def __init__(self, app):
+        """
+        Initializes the middleware.
+
+        Args:
+            app: The FastAPI application instance.
+        """
         super().__init__(app)
+        self.ROUTES_TO_EXCLUDE = ["/", "/status", "/docs", "/redoc"]
```

Comment:
we don't have to include "/" here as it is handled on the frontend. better to leave it to avoid breaking code